src/js/containers/PerformancePage/PerformancePage.js

```diff
@@ -11,6 +11,7 @@ import BrowserCacheTTLCard from '../../containers/BrowserCacheTTLCard/BrowserCac
 import DevelopmentModeCard from '../../containers/DevelopmentModeCard/DevelopmentModeCard';
```

Comment:
line 23
```
let isEmpty = _.isEmpty(zoneSettings[activeZoneId]);
```
Now needs to check if the plugin settings have loaded too, use the `getPluginSettingsForZoneId(zoneId, state)` to see if the state tree is empty.